deploy/charts/rook-ceph/templates/securityContextConstraints.yaml

```diff
@@ -0,0 +1,91 @@
+# scc for the Rook and Ceph daemons
+# for creating cluster in openshift
+{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: rook-ceph
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostPID: false
+# set to true if running rook with host networking enabled
+allowHostNetwork: false
+# set to true if running rook with the provider as host
+allowHostPorts: false
+priority:
+allowedCapabilities: ["MKNOD"]
+allowHostIPC: true
+readOnlyRootFilesystem: false
+# drop all default privileges
+requiredDropCapabilities: ["All"]
+defaultAddCapabilities: []
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: MustRunAs
+fsGroup:
+  type: MustRunAs
+supplementalGroups:
+  type: RunAsAny
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - hostPath
+  - persistentVolumeClaim
+  - projected
+  - secret
+users:
+  # A user needs to be added for each rook service account.
+  - system:serviceaccount:{{ .Release.Namespace }}:rook-ceph-system
+  - system:serviceaccount:{{ .Values.namespace }}:default
```

Comment:
We need to split the SCCs up differently. The operator helm chart does not know the namespace of the cluster (or multiple clusters) that will be installed. 
1. Remove lines 41-44 from this SCC
2. Create a new SCC in the **rook-ceph-cluster** templates that is exactly the same as this one, except:
   - Name the SCC something like: `name: rook-cluster-{{ .Release.Namespace }}`
   - Remove line 40 and keep lines 41-44

Reply:
Thanks, 
Yaa we need to re-name and have an extra SCC